src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -163,34 +151,13 @@ public void Prepare(CancellationToken token)
 
                 Debug.Assert(sourceControlledFileSystem.FilesExistOrQueuedToBeWritten(info.SolutionLevelFilePaths), "Expected solution items to be queued for writing");
             }
-
-            foreach (var project in projects)
-            {
-                if (token.IsCancellationRequested)
-                {
-                    return;
-                }
-
-                var projectBinder = projectBinderFactory.Get(project);
```

Comment:
After the last commit, `IProjectBinderFactory.Get(...)` always returned `null`.

This class didn't do anything else with the supplied list of `projects` except pass them to the factory to get binders, so I removed all of the code that referenced `projects` from this class too.